hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -208,25 +204,58 @@ class AzkabanHelper {
   }
 
   /**
-   * Makes sure that the pre-requisites of using the azkaban tasks are satisfied.
+   * Configures the fields for azkabanUploadTask.
    *
    * @param azkProject The AzkabanProject
+   * @return boolean to save the entered fields to .azkabanPlugin.json
    */
-  static void configureTask(AzkabanProject azkProject) {
+  static boolean configureTask(AzkabanProject azkProject) {
     def console = System.console();
     if (console == null) {
       throw new GradleException(
           "\nCannot access the system console. To use this task, explicitly set JAVA_HOME to the version specified in product-spec.json (at LinkedIn) and pass --no-daemon in your command.");
     }
 
-    if (!azkProject.azkabanUserName) {
-      azkProject.azkabanUserName = console.readLine("\nAzkaban user name: ");
-      logger.lifecycle("Run writeAzkabanPluginJson task and enter azkabanUserName field in .azkabanPlugin.json to automatically fetch the username.");
-    }
+    println("INTERACTIVE MODE:");
+    println("Current Azkaban project name : ${azkProject.azkabanProjName}");
+    println("Current Azkaban URL : ${azkProject.azkabanUrl}");
+    println("Current Azkaban username : ${azkProject.azkabanUserName}");
+    println("Current Azkaban ZipTask : ${azkProject.azkabanZipTask}");
+
+    try{
+      def input = console.readLine("\nWant to change any of the above? [y/n(default)] :");
+      if (input.toString() == 'y' || input.toString() == 'Y') {
+        input = console.readLine("New Azkaban project name (leave blank to restore current):");
+        if (input != null && !input.isEmpty()) {
+          azkProject.azkabanProjName = input.toString();
+        }
+
+        input = console.readLine("New Azkaban URL (leave blank to restore current):");
+        if (input != null && !input.isEmpty()) {
+          azkProject.azkabanUrl = input.toString();
+        }
+
+        input = console.readLine("New Azkaban username (leave blank to restore current):");
+        if (input != null && !input.isEmpty()) {
+          azkProject.azkabanUserName = input.toString();
+        }
 
-    if (!azkProject.azkabanProjName) {
-      azkProject.azkabanProjName = console.readLine("\nAzkaban project name: ");
-      logger.lifecycle("Run writeAzkabanPluginJson task and enter azkabanProjName field in .azkabanPlugin.json to automatically fetch the azkaban project name.");
```

Comment:
I have one big ask about the zip task. I'll write a separate comment about it in the GitHub conversation.